robottelo/api/utils.py

```diff
@@ -797,3 +798,10 @@ def disable_syncplan(sync_plan):
     sync_plan.enabled = False
     sync_plan = sync_plan.update(['enabled'])
     assert sync_plan.enabled is False
+
+
+def validate_filepath(sat_obj, org, file_names=['*.json', '*.tar.gz']):
```

Comment:
I suggest renaming this function to include Pulp in its name as it searches only in the Pulp export dir, or allowing the user to specify where to search for files.